.github/dependabot.yml

```diff
@@ -3,7 +3,9 @@ updates:
 
   # GitHub actions
   - package-ecosystem: "github-actions"
-    directory: "/"
+    directories:
+      - "/"
+      - "/.github/actions/*"
```

Comment:
```suggestion
      - "/.github/actions/*"
      - "/.github/workflows/*"
```

Reply:
> as well as the action.yml/action.yaml file from the root directory.

IIUC, it will not consider nested folders, as we have in our case:

- `.github/workflows/bootstrap/action.yml`

Maybe I'm wrong, but, I recall we had to use the folder to make this working.

Reply:
Thanks for double checking it 🙇